src/executors/golang-executor.yml

```diff
@@ -0,0 +1,5 @@
+description: "CircleCI provided docker image with Golang installed. see: https://hub.docker.com/r/circleci/golang/"
+docker:
+  - image: circleci/golang:1.12
```

Comment:
You might up it to 1.13 or 1.14 (due out soon).

Reply:
Done: 
66c36d0